src/Models/VarianceDissipationComputations/VarianceDissipationComputations.jl

```diff
@@ -97,11 +97,6 @@ end
 
 function (ϵ::VarianceDissipation)(model)
 
-    # Check if the model is using a QuasiAdamsBashforth2 time stepper
-    if !(model.timestepper isa QuasiAdamsBashforth2TimeStepper)
-        throw(ArgumentError("VarianceDissipation is only supported for QuasiAdamsBashforth2 time-stepping."))
-    end
-
```

Comment:
From the PR comment I understood it works now for QuasiAdamsBashforth2 + SplitRungeKutta3Timestepper but not yet for RK3? Do we need a different validation?

Also let's modify the docstring compat note to say which time steppers are available?